CONTRIBUTING.md

```diff
@@ -0,0 +1,106 @@
+
+The body of the pull request should contain enough description about what your contribution does together with any justification/reasoning. You should include references to any discussions (for example other tickets or mailing list discussions). You can add more commits to your pull request by committing them locally and pushing them to your branch and/or fork until you have satisfied feedback.
+
+With content submissions, in-depth explanations and linking to sources are needed, without exception, in cases of claims concerning the history of Dogecoin and text, images, or other media associated with it.
+
+## Pull Request Philosophy
+
+Pull Requests should always be focused. For example, a pull request could add an article, fix a bug, or refactor code; but not a mixture. Please avoid submitting pull requests that attempt to do too much, are overly large, or overly complex as this makes review difficult.
+
+It is particularly crucial to keep frontend content edits - for example, the creation of new articles or edits to existing ones - separated from code edits and bug fixes.
+
+Just like with code, the history of articles, faqs, and informative sections of the website must be preserved and easy to track and revert.
+
+
+### Features
+
+When adding a new feature, thought must be given to the long-term maintenance that feature may require after inclusion. Before proposing a new
+feature or section to the website that will require maintenance, please consider if you are willing to maintain it (including bug fixing). If features get orphaned with no maintainer in the future, they may be removed.
+
+
+## "Decision Making" Process
+
+Whether a pull request is merged and published on the website rests with the repository maintainers collaborating with contributors.
+
+Maintainers will take into account the consensus among contributors. Maintainers cannot bypass, at any moment, the need for peer reviews. 
+
+In case of content submissions, maintainers and contributors will consider the quality, soundness, and impartiality of the content, alongside appropriate documentation and factual information corroborating statements within the submitted content.
+
+Maintainers will take into consideration if a submission is in line with the general principles of Dogecoin.
+
+In general, all pull requests must:
+
+  - Have a clear use case, fix a demonstrable bug, or provide content useful to the Dogecoin community;
+  - Not be an expression of particular interests of companies, organizations, or individuals;
+  - Be focused;
+  - Be peer-reviewed;
+  - In case of code changes, have gone through functional tests;
+  - Follow code and content style guidelines;
+  - Not break the website's building process;
+
+The following contributions must come with significant discussion, happening in public and accessible to all, before
+approval and merge:
+
+- Major structural changes
+- Major content changes
+- Linking to third-party resources
```

Comment:
About this. Will the discussing be on Gigthub? 
How significant should be the discussion?

Example, Wallets that links to third-party resources there are already some pending for review for some time, how should it be handle if nobody else discuss about it?

Reply:
We certainly do not have enough activity on this Github, but I think that when it comes to wallets the only thing you can do is to wait for others to share their opinion. 

Very very difficult to vouch for a wallet, for example, when they have not been audited by reliable third parties and still do not have a long enough track record. While you have disclaimers in place in the wallet list, I do feel queasy saying "that's OK". You will see the resources I added in the "resources" section (current version was really just little more than a stub) were mostly relatively risk-free picks.

Facing the same issue on Twitter right now, with some wallet manufacturers offering to send me their (very expensive) wallets, with some interesting features regarding backup management, but that I would not be able to review properly with my means. Too big of a responsibility.

Reply:
Understood. Much thanks :)